lib/routes/routePUT.js

```diff
@@ -120,6 +120,14 @@ export default function routePUT(request, response, log, utapi) {
         // PUT object copy
         // if content-md5 is not present in the headers, try to
         // parse content-md5 from meta headers
+
+        if (request.headers['content-md5'] === '') {
```

Comment:
Yes, sorry @alexandre-merle, we talked about that we Lauren but I did not update the status.
@alexandre-merle you are right, there are more cases like: 
- unreadable MD5 `'Content-MD5': '\x07'`
- invalid MD5 `'Content-MD5':'YWJyYWNhZGFicmE='`